src/core/api.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+const ApiService = {
+  init() {
+    axios.defaults.baseURL = "http://react-cdp-api.herokuapp.com/";
```

Comment:
According to "12-factor app" (https://12factor.net/), such things should be put under configs of the applications to allow configuration of the application using environment variables.
Consider extracting this URL and putting it as API_URL under config directory.